src/backend/distributed/sql/udfs/create_distributed_function/latest.sql

```diff
@@ -1,7 +1,12 @@
-CREATE OR REPLACE FUNCTION create_distributed_function(function_name regprocedure)
+CREATE OR REPLACE FUNCTION create_distributed_function(function_name regprocedure,
```

Comment:
Should this change be in a separate commit or PR? This doesn't seem relevant to other PRs without an actual implementation.

We should also make sure to use qualified names in these scripts or move them into the upgrade script such that the search path is clear.

Reply:
Ok, makes sense. We can discuss the API on another PR.